### DISCUSSION TITLE
¿Qué tan única es la ID de los pingüinos?

### DISCUSSION THREAD

**Discussion by Utmite** [IIC2133-PUC/2024-2#103] [Tarea 1] [answered]
Hola, buenas.

Me pregunto si la ID de los pingüinos es única dentro de los sectores o si es única para todos los pingüinos.

Muchas gracias.

**Comment by tomastrivino** [comment, marked as answer]
Hola,
cada pingüino tiene un ID asociado. Nunca habrán dos pingüinos con el mismo ID.
